streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/ingress/IngressStrategy.scala

```diff
@@ -47,11 +47,15 @@ trait IngressStrategy {
     }
   }
 
-  def buildIngressAnnotations(clusterId: String): Map[String, String] = {
+  def buildIngressAnnotations(clusterId: String, namespace: String): Map[String, String] = {
     val annotations = Map(
       "nginx.ingress.kubernetes.io/rewrite-target" -> "/$2",
       "nginx.ingress.kubernetes.io/proxy-body-size" -> "1024m",
-      "nginx.ingress.kubernetes.io/configuration-snippet" -> ("rewrite ^(/" + clusterId + ")$ $1/ permanent;")
+      "nginx.ingress.kubernetes.io/configuration-snippet" -> ("rewrite ^(/" + clusterId + ")$ $1/ permanent;" + String
+        .format(
```

Comment:
It can be made more concise by modifying it to:
```scala
"nginx.ingress.kubernetes.io/configuration-snippet" -> s"rewrite ^(/ $clusterId )$$ $$1/ permanent; sub_filter '<base href=\"./\">' '<base href=\"/$namespace/$clusterId/\">'; sub_filter_once off;" 
```

Reply:
THKS,  already fix it. PTAL